src/cr/cube/dimension.py

```diff
@@ -937,7 +941,7 @@ def is_insertion(self):
 
         Unconditionally True for _Subtotal objects.
         """
-        return True
+        return True  # pragma: no cover
```

Comment:
This needs to be tested IMO. The rationale - if you put a `raise` here - it will destroy exporter tests.

A bit more context: you need to check if some rows or columns from the exporter will need to be *bold*. So you need to be able to tell if they're insertions or not. An integration test will preserve this context, but you don't have to do that, a simple unit test will prevent regression.

Reply:
I changed an integration test here: 2ecc9795f88a2ef833b9948da5eadd720d2d2ef7